Description: Upstream changes introduced in version 0.65-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 dhex (0.65-1) unstable; urgency=low
 .
   * Initial release. (Closes: #613929)
 .
 The person named in the Author field signed this changelog entry.
Author: Gürkan Sengün <gurkan@phys.ethz.ch>
Bug-Debian: http://bugs.debian.org/613929

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- dhex-0.65.orig/Makefile
+++ dhex-0.65/Makefile
@@ -5,7 +5,7 @@ CFLAGS=		-O3 -Wall -std=c99
 #CFLAGS+= -ffunction-sections -fdata-sections
 #LDFLAGS+= --gc-sections 
 LIBS=		-lncurses
-DESTDIR=	/usr/local/
+#DESTDIR=	/usr/
 
 OFILES=buffers.o \
 	configfile.o \
@@ -28,11 +28,11 @@ dhex:	$(OFILES)
 
 install:all
 	strip dhex
-	cp dhex $(DESTDIR)/bin
-	cp dhex.1 $(DESTDIR)/share/man/man1
-	cp dhexrc.5 $(DESTDIR)/share/man/man5
-	cp dhex_markers.5 $(DESTDIR)/share/man/man5
-	cp dhex_searchlog.5 $(DESTDIR)/share/man/man5
+	cp dhex $(DESTDIR)/usr/bin/
+	cp dhex.1 $(DESTDIR)/usr/share/man/man1
+	cp dhexrc.5 $(DESTDIR)/usr/share/man/man5
+	cp dhex_markers.5 $(DESTDIR)/usr/share/man/man5
+	cp dhex_searchlog.5 $(DESTDIR)/usr/share/man/man5
 
 	
 
